components/sysutils/phoronix-test-suite/manifests/sample-manifest.p5m

```diff
@@ -0,0 +1,7700 @@
+file path=usr/share/applications/phoronix-test-suite.desktop
+file path=usr/share/doc/phoronix-test-suite/AUTHORS
+file path=usr/share/doc/phoronix-test-suite/COPYING
+file path=usr/share/doc/phoronix-test-suite/ChangeLog
+file path=usr/share/doc/phoronix-test-suite/includes/pts-documentation.css
+file path=usr/share/doc/phoronix-test-suite/includes/pts-documentation.png
+file path=usr/share/doc/phoronix-test-suite/phoronix-test-suite.html
+file path=usr/share/doc/phoronix-test-suite/phoronix-test-suite.md
+file path=usr/share/doc/phoronix-test-suite/phoronix-test-suite.pdf
+file path=usr/share/doc/phoronix-test-suite/stubs/00_overview.html
+file path=usr/share/doc/phoronix-test-suite/stubs/00_user_options.html
+file path=usr/share/doc/phoronix-test-suite/stubs/00_zmodule_options.html
+file path=usr/share/doc/phoronix-test-suite/stubs/01_install.html
+file path=usr/share/doc/phoronix-test-suite/stubs/01_windows.html
+file path=usr/share/doc/phoronix-test-suite/stubs/02_external_dependencies.html
+file path=usr/share/doc/phoronix-test-suite/stubs/40_configuration.html
+file path=usr/share/doc/phoronix-test-suite/stubs/41_configuration_file.html
+file path=usr/share/doc/phoronix-test-suite/stubs/42_env_vars.html
+file path=usr/share/doc/phoronix-test-suite/stubs/50_general_information.html
+file path=usr/share/doc/phoronix-test-suite/stubs/55_virtual_suites.html
+file path=usr/share/doc/phoronix-test-suite/stubs/59_component_testing.html
+file path=usr/share/doc/phoronix-test-suite/stubs/60_phoromatic.html
+file path=usr/share/doc/phoronix-test-suite/stubs/67_offline.html
+file path=usr/share/doc/phoronix-test-suite/stubs/68_offline_additions.html
+file path=usr/share/doc/phoronix-test-suite/stubs/99_development_credits.html
+file path=usr/share/doc/phoronix-test-suite/stubs/downloads.html
+file path=usr/share/doc/phoronix-test-suite/stubs/readme-basics.txt
+file path=usr/share/doc/phoronix-test-suite/test-profile-creation.md
+file path=usr/share/icons/hicolor/48x48/apps/phoronix-test-suite.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/application-x-openbenchmarking.png
+file path=usr/share/man/man1/phoronix-test-suite.1
+file path=usr/share/mime/packages/openbenchmarking-mime.xml
+file path=usr/share/phoronix-test-suite/deploy/deb-package/build-package-deb.php
+file path=usr/share/phoronix-test-suite/deploy/deployments.md
+file path=usr/share/phoronix-test-suite/deploy/docker/ubuntu-pts-docker-build.sh
+file path=usr/share/phoronix-test-suite/deploy/phoromatic-initd/phoromatic-client
+file path=usr/share/phoronix-test-suite/deploy/phoromatic-rcd/phoromatic-client
+#file path=usr/share/phoronix-test-suite/deploy/phoromatic-systemd/phoromatic-client.service
+#file path=usr/share/phoronix-test-suite/deploy/phoromatic-systemd/phoromatic-server.service
+#file path=usr/share/phoronix-test-suite/deploy/phoromatic-systemd/phoronix-result-server.service
+file path=usr/share/phoronix-test-suite/deploy/phoromatic-upstart/phoromatic-client.alt.conf
```

Comment:
Please don't edit sample-manifest.p5m. It is a generated file and should be used as a base for the real manifest (which can be edited) and as a reference when building on other machines, other times, or other versions.

Reply:
Okay, I was trying to avoid installing (unnecessary) systemd files, but we can always figure that out later with a more elegant solution

Reply:
> Okay, I was trying to avoid installing (unnecessary) systemd files, but we can always figure that out later with a more elegant solution

The content of `sample-manifest.p5m` does not affect what gets packaged and/or installed.  The manual change of `sample-manifest.p5m` is always no-op and counterproductive (as described above by @AndWac).